res/css/style.css

```diff
@@ -602,20 +603,21 @@ body, #root, .profileViewer {
 .tabBarContainer {
   display: flex;
   flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: center;
   background: var(--grey-10);
   position: relative;
   border: solid var(--grey-30);
   border-width: 1px 0 1px;
 }
 
 .tabBarTabWrapper {
-  margin: 0;
   padding: 0 0.5px;
   list-style: none;
   display: flex;
   flex-flow: row nowrap;
   margin: 0 -1px;
-  max-width: 100%;
+  min-width: 0; /* This makes the tab container actually shrinkable below min-content */
```

Comment:
Eventually we should move these styles to a separate CSS file and reorder them a bit...

Reply:
Might be worth a look, yes. Wanna open an issue for this?

Reply:
Sure, I filed #1119